test/run.sh

```diff
@@ -275,6 +281,15 @@ function run_acceptance_replication_tests() {
   done
 }
 
+function run_acceptance_maintenance_tests() {
```

Comment:
I would make this as a new generic option to run distinct packages in `test/acceptance` package, now this test is limited only to your package.

I would implement it the same way we have `--only-module-{moduleName}` param implemented, just in your case it would be `--only-acceptance-{packageName}` so that your test could be run with `--only-acceptance-maintenance_mode` modifier, wdyt?